src/conf_mode/protocols_mld.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2023 VyOS maintainers and contributors
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 or later as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+
+from ipaddress import IPv6Address
+from sys import exit
+
+from vyos import ConfigError
+from vyos.config import Config
+from vyos.utils.process import process_named_running
+from vyos.utils.process import call
+from vyos.template import render
+from signal import SIGTERM
+
+from vyos import airbag
+airbag.enable()
+
+# Required to use the full path to pim6d, in another case daemon will not be started
+pim6d_cmd = f'/usr/lib/frr/pim6d -d -F traditional --daemon -A 127.0.0.1'
```

Comment:
This is for IPv6, correct? Or should pim for IPv4 be used?

Reply:
Yes, this is for IPv6. Only pim6d is needed for MLD.